src/3rd party/luajit-2/src/lj_debug.c

```diff
@@ -440,7 +440,8 @@ int lj_debug_getinfo(lua_State *L, const char *what, lj_Debug *ar, int ext)
   GCfunc *fn;
   if (*what == '>') {
     TValue *func = L->top - 1;
-    api_check(L, tvisfunc(func));
+    if (!tvisfunc(func))
+	  return 0;
```

Comment:
It would be good to replace this tab with 4 spaces to be consistent :)
```suggestion
      return 0;
```